docs/manual/howtos/exchange.rst

```diff
@@ -12,60 +12,59 @@
 .. code:: C++
 
 	dds::domain::DomainParticipant participant(domain::default_id());
 
-, or, if you want to have more control over the process, select your own ID:
+or, to have more control over the process, select your own ID:
 
 .. code:: C++
 
 	dds::domain::DomainParticipant participant(123456);
 
-The main part here is that you have the same ID on the reading side as the writing, because they won't be visible to eachother otherwise.
+The main point is that you must have the same ID on both the reading side and the writing side, otherwise, they can not see each other.
 ???Explain more about setting QoSes???
 
 Topic
 -----
 
-A Topic is a subsection of a DDS Domain which allows exchange of data of a specific type which adheres to certain restrictions on the exchange before exchange can occur. 
+A Topic is a subsection of a DDS Domain that enables exchange of data of a specific type, and that complies with certain restrictions on the exchange before exchange can occur. 
 A Topic is identifiable by:
 
-- a Name
+- Name
 	- identifies the topic on the Domain
-	- has to be unique on the Domain
-- a Type
+	- must be unique on the Domain
+- Type
 	- the type of data being exchanged
 	- is the template parameter of the dds::topic::Topic class
-- a Quality of Service
-	- determines the restrictions on the exchange occurring
+- Quality of Service
+	- determines the restrictions on the exchange
 	- this is an optional parameter, and can be derived from fallbacks to the participant or defaults
 
-A Topic is for exchanging data of the Type Data_Type is created on the DomainParticipant participant in the following manner:
+A Topic is for exchanging data of the Type Data_Type. The following shows how a topic is created on the DomainParticipant participant:
 
 .. code:: C++
 
 	dds::topic::Topic<Data_Type> topic(participant, "DataType Topic");
 
-The data type of the topic is generated from the user's IDL files by using CycloneDDS's idlc generator with the idlcxx library.
-Using types other than those generated from idlc+idlcxx in the template will not have the prerequisite traits and therefore not result in working code.
+To generate the data type of the topic from the user's IDL files, use CycloneDDS's idlc generator (with the idlcxx library).
+Using types other than those generated from idlc+idlcxx in the template does not have the prerequisite traits, and therefore does not result in working code.
 
 Publishers
 ----------
 
-A Publisher is a producer of data on a Domain. It uses the DomainParticipant to gain access to the Domain and is created using it.
-A Publisher allows the DataWriters associated with it to share the same behaviour, such as:
+A Publisher is a producer of data on a Domain. It uses the DomainParticipant to gain access to the Domain and is created using it. That is, the Publisher inherits the Domain from its parent class DomainParticipant. A Publisher allows the DataWriters associated with it to share the same behaviour, for example:
```

Comment:
"Inherit" might not be the best choice of wording here, as this has a specific meaning inside C++. Maybe use "is passed down" or something similar?

Reply:
changed to "passes down"